indico/queries/workflow.py

```diff
@@ -1,49 +1,86 @@
 from typing import List
 
-from indico.client.request import (
-    GraphQLRequest,
-    HTTPMethod,
-    HTTPRequest,
-    RequestChain,
-)
+from indico.client.request import GraphQLRequest, RequestChain
 from indico.errors import IndicoError
 from indico.queries.storage import UploadDocument
-from indico.types.jobs import Job
-from indico.types.workflow import Workflow
+from indico.types import Job, Workflow
 
 
-class ListWorkflowsForDataset(GraphQLRequest):
+class ListAllWorkflows(GraphQLRequest):
```

Comment:
Lets name this to ListWorkflows. We can drop ListWorkflowsForDataset (I'll make this version 4.0 of the client so breaking change is ok+ I doubt anyone was using ListWorkflowForDataset)